doc/angelscript/game.h

```diff
@@ -288,7 +288,7 @@ class game
 	 * @param instanceName The unique name that you chose when spawning this object
 	 * @param pos The position where the object should be moved to
 	 */
-	void moveObjectVisuals(const Ogre::String& instanceName, const Ogre::Vector3& pos);
+	void moveObjectVisuals(const string instanceName, const vector3 pos);
```

Comment:
Is this necessary? I think the coding guidelines mentioned to use Ogre::String and Ogre::Vector3 but I can't this information anymore.